src/code/z_rcp.c

```diff
@@ -1,143 +1,1430 @@
 #include <ultra64.h>
 #include <global.h>
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_rcp/func_80093370.s")
+Gfx sSetupDL[] = {
```

Comment:
Could sSetupDL possibly be a Gfx[][6] to avoid indexing by 6 * idx?

Reply:
Unfortunately it can't, that's what I went for originally but Gfx_CallSetupDLImpl won't match if it's a Gfx[][6].